### ISSUE TITLE
Reset amount ignored

### ISSUE THREAD

**Issue by N0tMyFaultOG** [BentoBoxWorld/BSkyBlock#271]
#### Describe the bug

It appears that resets applied with `/bsbadmin resets` are not being considered when resetting an island.

#### Steps to reproduce the behavior

Reset your island until you have no resets left and apply resets manually.

#### Expected behavior

Bsbadmin limits should overwrite total limits.

#### Screenshots and videos (Optional)

![](https://i.imgur.com/HR3cDvZ.png)    
I should have 40 resets left, yet `/is resets` does not seem to find my other resets.

##### BentoBox Version (Mandatory)

![](https://i.imgur.com/zkI9o7L.png)

**Comment by tastybento** [comment]
I'm not sure what specific commands you are running, but in BSkyBlock resets are a count of how many times the player has reset their island, not a count of how many resets they have left.

Here is the command help:
```
[17:14:50 INFO]: =========== BSkyBlock help ===========
[17:14:50 INFO]: /bsbadmin resets: edit resets of the players
[17:14:50 INFO]: /bsbadmin resets set <player> <resets>: sets how many time this player reset his island
[17:14:50 INFO]: /bsbadmin resets reset <player>: resets how many time this player reset his island to 0
[17:14:50 INFO]: /bsbadmin resets add <player> <resets>: adds to how many times this player reset his island
[17:14:50 INFO]: /bsbadmin resets remove <player> <resets>: removes from how many times this player reset his island
[17:14:50 INFO]: =================================
```

So, for me, here is my status:
```
> bsb info tastybento
[17:17:00 INFO]: ========== Island Info ============
[17:17:00 INFO]: UUID: BSkyBlock5381f0c0-89f9-4a3d-8db9-33642300ac9e
[17:17:00 INFO]: Owner: tastybento (5988eecd-1dcd-4080-a843-785b62419abb)
[17:17:00 INFO]: Last login: Wed Mar 04 14:46:42 PST 2020
[17:17:00 INFO]: Deaths: 0
[17:17:00 INFO]: Resets: 3 (Max: 3)
```
I have used 3 resets and the max allowed is 3.

Now I will remove 1 reset from my count so I can reset one more time:

```
> bsb resets remove tastybento 1
[17:18:24 INFO]: Successfully removed 1 resets to tastybento, decreasing the total to 2 resets.
```
I can now reset the island again.

So, in summary - the reset number is how many times the player has reset their island. The maximum number of resets a player can have is configured in the config.yml file. If you want to allow a player to have the maximum again, then rung the `bsb resets reset <player>` command. The other commands are just for fine-tuning.

**Comment by N0tMyFaultOG** [comment]
My bad, the command works that way, but I expected it to work vice versa.
